lte/gateway/c/session_manager/ShardTracker.h

```diff
@@ -13,39 +13,37 @@
 #pragma once
 #include <string>
 #include <vector>
+#include <set>
 
 namespace magma {
 
 /* Shards represent groups of UEs placed into buckets of
 a certain size, to make polling more manageable*/
 class ShardTracker {
-  ShardTracker();
-
+ public:
   /**
    * Add UE to shards based on availability
    * @return index(shard id) where UE was placed
-   * TODO(veshkemburu): Store IMSI as well for easier subscriber reallocation
```

Comment:
Add documentation on what would happen if the UE already has a shardID. Does it return the existing shardID?